messagebus/src/vespa/messagebus/network/rpcnetwork.cpp

```diff
@@ -86,8 +86,10 @@ RPCNetwork::SendContext::handleVersion(const vespalib::Version *version)
         }
     }
     if (shouldSend) {
-        _net.send(*this);
-        delete this;
+        _net.getExecutor().execute(vespalib::makeLambdaTask([this]() {
+            _net.send(*this);
```

Comment:
two things are not obvious to me:
(1) this is the only place messages are sent
(2) this will not lead to send being called on a destructed rpc network